Add Biomedical Information Extraction

caufieldjh opened the pull request:
<!-- Congrats on creating an Awesome list! 🎉 -->


<!-- Please fill in the below placeholders -->

**https://github.com/caufieldjh/awesome-bioie#readme**

**How to extract information from unstructured biomedical data and text.**

Pull requests reviewed:
#1615 
#1609 

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- **Don't waste my time.** Do a good job, adhere to all the guidelines, and be responsive.
- **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).**
	Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration.
- You have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a title in the format `Add Name of List`.
	- ✅ `Add Swift`
	- ✅ `Add Software Architecture`
	- ❌ `Update readme.md`
	- ❌ `Add Awesome Swift`
	- ❌ `Add swift`
	- ❌ `Adding Swift`
	- ❌ `Added Swift`
- Your entry here should include a short description about the project/theme of the list. **It should not describe the list itself.** The first character should be uppercase and the description should end in a dot. It should be an objective description and not a tagline or marketing blurb.
	- ✅ `- [iOS](…) - Mobile operating system for Apple phones and tablets.`
	- ✅ `- [Framer](…) - Prototyping interactive UI designs.`
	- ❌ `- [iOS](…) - Resources and tools for iOS development.`
	- ❌ `- [Framer](…)`
	- ❌ `- [Framer](…) - prototyping interactive UI designs`
- Your entry should be added at the bottom of the appropriate category.
- The suggested Awesome list complies with the below requirements.


## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- Don't open a Draft / WIP pull request while you work on the guidelines. A pull request should be 100% ready and should adhere to all the guidelines when you open it.
- Run [`awesome-lint`](https://github.com/sindresorhus/awesome-lint) on your list and fix the reported issues. If there are false-positives or things that cannot/shouldn't be fixed, please [report it](https://github.com/sindresorhus/awesome-lint/issues/new).
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- ✅ `Mobile operating system for Apple phones and tablets.`
	- ✅ `Prototyping interactive UI designs.`
	- ❌ `Resources and tools for iOS development.`
	- ❌ `Awesome Framer packages and tools.`
- It's the result of hard work and the best I could possibly produce.
	**If you have not put in considerable effort into your list, your pull request will be immediately closed.**
- The repo name of your list should be in lowercase slug format: `awesome-name-of-list`.
	- ✅ `awesome-swift`
	- ✅ `awesome-web-typography`
	- ❌ `awesome-Swift`
	- ❌ `AwesomeWebTypography`
- The heading title of your list should be in [title case](https://capitalizemytitle.com/) format: `# Awesome Name of List`.
	- ✅ `# Awesome Swift`
	- ✅ `# Awesome Web Typography`
	- ❌ `# awesome-swift`
	- ❌ `# AwesomeSwift`
- Non-generated Markdown file in a GitHub repo.
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate. Please search for existing submissions.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Includes a project logo/illustration whenever possible.
	- Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- **The image should be high-DPI.** Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
		- Can be placed centered if the list has a centered graphics header.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- **If you use a license badge, it should be SVG, not PNG.**
	- To verify that you've read all the guidelines, please add comment with just the word `unicorn`.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Doesn't include an `Inspired by awesome-foo` or `Inspired by the Awesome project` kinda link at the top of the readme. The Awesome badge is enough.

**Go to the top and read it again.**

caufieldjh commented:
unicorn

caufieldjh commented:
I got a couple false positive broken links out of the linter; related to the issues mentioned here: https://github.com/sindresorhus/awesome-lint/issues/89

dialex commented:
Should not be merged yet 🛑

- The PR title should be `Add Biomedical Information Extraction`, per the guidelines of this repo

sindresorhus commented:
I don't think you should have both the text header and the image header. They both have the same content, so I would remove the text header, and move the image header to the top.

---

```
❯ awesome-lint https://github.com/caufieldjh/awesome-bioie
✖ Linting

  README.md:57:25
  ✖   1:1    Missing Awesome badge after the main heading                                                                              remark-lint:awesome-badge
  ✖  44:3    ToC item "Protein-protein Interaction Annotated Corpora" link "#Protein-protein-interaction-annotated-corpora" not found  remark-lint:awesome-toc
  ✖  57:25   "’" is used without matching "‘"                                                                                          remark-lint:match-punctuation
  ✖  84:241  Should not repeat "."                                                                                                     remark-lint:no-repeat-punctuation
  ✖  84:242  Should not repeat "."                                                                                                     remark-lint:no-repeat-punctuation

  5 errors
```

https://github.com/sindresorhus/awesome-lint

`#Protein-protei` => `#protein-protei` to fix the second item.


---

Can you fix the merge conflict?

---

> Includes a project logo/illustration whenever possible.
>> **The image should be high-DPI. Set it to maximum half the width of the original image.**

caufieldjh commented:
OK - made the following changes:
* Text header removed - just have the image header now.
* Linter errors resolved - only error is now _Missing main list heading_ since there's only the image header.
* Fixed the typo in the table of contents.
* Resolved merge conflict.
* Replaced the header image with a new version, now at requested resolution.

Please let me know if any further edits would help!

sindresorhus commented:
You need to add a `<br>` after `<img src="https://github.com/caufieldjh/awesome-bioie/blob/master/images/abie_head.png" alt="Awesome BioIE Logo"/>` as image elements are inline.

---

Don't put empty lines between list elements.

sindresorhus commented:
Tweet: https://twitter.com/awesome__re/status/1199347510097010688

caufieldjh commented:
> You need to add a `<br>` after `<img src="https://github.com/caufieldjh/awesome-bioie/blob/master/images/abie_head.png" alt="Awesome BioIE Logo"/>` as image elements are inline.
> 
> Don't put empty lines between list elements.

Done and done!

Thanks!
